Formula/n/nesfab.rb

```diff
@@ -0,0 +1,51 @@
+class Nesfab < Formula
+  desc "Programming language that targets the Nintendo Entertainment System"
+  homepage "https://pubby.games/nesfab.html"
+  # for this version only, point to a specific commit. post-1.6, this will point to a tagged release.
+  url "https://github.com/pubby/nesfab/archive/da18a43dc7b941cc4c56c949303aa37633fdc1b0.tar.gz"
+  version "1.6"
```

Comment:
We'll need to wait for a tagged release

Reply:
OK, I will ask upstream if they can do a small release so we have something to point to.